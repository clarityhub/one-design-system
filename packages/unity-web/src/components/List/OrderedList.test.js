import React from 'react';
import renderer from 'react-test-renderer';
import { ListItem, OrderedList } from './index';

describe('<OrderedList />', () => {
	it('renders items', () => {
		const node = (
			<OrderedList>
				<ListItem>First item</ListItem>
				<ListItem>Second item</ListItem>
				<ListItem>Third item</ListItem>
			</OrderedList>
		);
		const tree = renderer.create(node).toJSON();

		expect(tree).toMatchSnapshot();
	});
});