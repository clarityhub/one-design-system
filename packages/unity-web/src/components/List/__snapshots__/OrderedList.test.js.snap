// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderedList /> renders items 1`] = `
<ol
  className="css-1wwl84o-OrderedList e50npi0"
>
  <li
    className="css-8lg160"
  >
    First item
  </li>
  <li
    className="css-8lg160"
  >
    Second item
  </li>
  <li
    className="css-8lg160"
  >
    Third item
  </li>
</ol>
`;
